Taskfile

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+if [[ -z "${HISTORIC_TAG_DATA_PATH}" ]]; then
```

Comment:
What is historic tag data?  Existing tag?  Isn't this the value in the `default.env`?

Oh do you mean keeping all tags from the previous run and diff with new run to single out new tags since last run?

Reply:
Yes, actually `HISTORIC_TAG_DATA_PATH` represents the local path that will be synced with `DATA_DIR` (`data/`), via docker volume.
This path will then contain the historic tag values for each images, which will be used for diff.